save-plugins/warn-plugin/src/commonNonJsTest/kotlin/org/cqfn/save/plugin/warn/utils/ComparisonTest.kt

```diff
@@ -1,18 +1,13 @@
 package org.cqfn.save.plugin.warn.utils
 
-import org.cqfn.save.core.config.TestConfig
 import org.cqfn.save.core.files.createFile
-import org.cqfn.save.core.result.Pass
-import org.cqfn.save.plugin.warn.WarnPlugin
 import org.cqfn.save.plugin.warn.WarnPluginConfig
 
 import okio.FileSystem
 
 import kotlin.test.AfterTest
 import kotlin.test.BeforeTest
 import kotlin.test.Test
-import kotlin.test.assertEquals
-import kotlin.test.assertTrue
```

Comment:
Why there is no `unresolved reference` err after these changes?

Reply:
yes, bug